xmltree/xmltree.go

```diff
@@ -164,15 +164,24 @@ func (scope *Scope) Prefix(name xml.Name) (qname string) {
 func (scope *Scope) pushNS(tag xml.StartElement) []xml.Attr {
 	var ns []xml.Name
 	var newAttrs []xml.Attr
+	var targetNamespace string
+	hasXmlns := false
```

Comment:
Same as comment made below - can assume zero value of bool
```suggestion
	var hasXmlns bool
```

Reply:
updated